tox.ini

```diff
@@ -76,5 +76,17 @@ deps =
     pytest-operator
     -r{toxinidir}/requirements.txt
 commands =
-    pytest -v --tb native --log-cli-level=INFO -s {posargs}
+    pytest -k "bundle" -v --tb native --log-cli-level=INFO -s {posargs}
```

Comment:
**question** what does this "bundle" refer to? I'm not sure I undersdtand its usage